samples/core/XGBoost/xgboost_sample.py

```diff
@@ -23,7 +35,7 @@ def xgboost_pipeline():
         label_column=0,
         objective='reg:squarederror',
         num_iterations=200,
-    ).outputs['model']
+    ).set_memory_limit('1Gi').outputs['model']
```

Comment:
this fixes the flakiness in this pipeline, caused by OOM

Reply:
Is this an empirical number? A nitpick would be have a comment on any magic number.

Reply:
Good point! Let me add a comment.